scripts/remote_deployment/.ipc/config.toml

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
TODO for the future:

- Instead of having a templated config.toml, we should use `ipc-cli config init`. This already exists but it creates default entries which are stale (calibnet, mycelium testnet). Need to figure out how to solve this.
- `ipc-cli subnet create` should add an entry for the new subnet in the config.toml automatically.

Reply:
Captured in https://linear.app/interplanetary-consensus/issue/ENG-716/automated-calibration-deployment-improvements-related-to-configtoml
IMO not a high priority now.